scripts/build/utils/generate-package-json.ts

```diff
@@ -14,14 +14,15 @@ export async function generatePackageJsonFile(cwd: string, data: BuildEntries) {
   // Add the package.json file to the exports, so we can use it to `require.resolve` the package's root easily
   pkgJson.exports = {
     './package.json': './package.json',
+    ...data.extraOutputs,
```

Comment:
Instead of this structure, I'd propose that the `entries` map had an `unbuilt`-key, that didn't get build by the builder, but just copied directly from `entrypoint`. so `./src/my-unbuilt-file.js` -> `./dist/my-unbuilt-file.js`
then it would also have `exportEntries` automatically, because it's an entry like the others.